docs/source/blocks.rst

```diff
@@ -0,0 +1,146 @@
+======
+Blocks
+======
+
+
+On a page, a block is a chunk of content which sits in a column.
+
+With Glitter, a block is a combination of a Django model and a template file,
+registered with the block admin so it can be selected in frontend templates.
+
+
+Quick example
+=============
+
+This example block defines a ``HeadlineText`` model, with just one attribute
+named ``text`` in ``models.py``.
+
+.. code:: python
+
+    from django.db import models
+
+    from glitter.models import BaseBlock
+
+
+    class HeadlineText(BaseBlock):
+        text = models.TextField()
+
+
+To make this usable in the frontend, we need to register this block in
+``blocks.py``.
+
+.. code:: python
+
+    from glitter.blockadmin import blocks
+
+    from .models import HeadlineText
+
+
+    @blocks.register(HeadlineText, category='Common')
+    class HeadlineTextAdmin(blocks.BlockAdmin):
```

Comment:
Is this should be just `HeadlineText` ? 
Naming is not very consistent in the documentation https://github.com/blancltd/django-glitter/pull/50/files#diff-1e2bb462d1d00960bfcc1d917a5f45eeR75